src/generate/rust.rs

```diff
@@ -412,9 +410,7 @@ impl<'a> Continuation<'a> {
 
     #[cfg_attr(rustfmt, rustfmt_skip)]
     fn reify_as(&mut self, label: CodeLabel) {
-        let code = format!("
-            {} => {{{}
-            }}", label, self.to_inline());
+        let code = format!("{} => {{{}}}", label, self.to_inline());
```

Comment:
This is horrible, aaaaaaaaaaah!!!
(not your fault, just `{{` and `}}` are bad escapes)
We should just have a version of `put!` that's like `format!` tbh.